presto-product-tests/README.md

```diff
@@ -89,16 +89,29 @@ following command:
     presto-product-tests/bin/run_on_docker.sh <profile> -x quarantine,big_query,mysql_connector,postgresql_connector
     ```
 
-where ``<profile>`` is one of either:
+where [profile](#profile) is one of either:
 - **multinode** - pseudo-distributed Hadoop installation running on a
  single Docker container and a distributed Presto installation running on
  multiple Docker containers. For multinode the default configuration is 1 coordinator and 1 worker.
-- **singlenode** - pseudo-distributed kerberized Hadoop installation running on a
- single Docker container and a single node installation of kerberized Presto also running
- on a single Docker container.
-- **singlenode-kerberized** - pseudo-distributed Hadoop installation running on a
+- **singlenode-simple-impersonation** - pseudo-distributed Hadoop installation running on a
+ single Docker container and a single node installation of Presto also running
+ on a single Docker container. Presto impersonates the user who is running the query when accessing HDFS.
+ The user specified by -DHADOOP_USER_NAME in jvm.config must be allowed to impersonate this user.
+- **singlenode-simple-no-impersonation** - pseudo-distributed Hadoop installation running on a
```

Comment:
Make sure that after renaming environments you haven't left any references to the old names in documentation and in the `.travis.yaml`